vault/identity_store.go

```diff
@@ -329,6 +329,10 @@ func (i *IdentityStore) CreateOrFetchEntity(alias *logical.Alias) (*identity.Ent
 		return nil, fmt.Errorf("invalid mount accessor %q", alias.MountAccessor)
 	}
 
+	if mountValidationResp.MountLocal {
```

Comment:
Shouldn't this happen later so that existing values are returned?

Reply:
It wouldn't matter. This function will not be called for the new tokens.